### DISCUSSION TITLE
increase Latex editor input text size?

### DISCUSSION THREAD

**Discussion by NelkonParker** [xournalpp/xournalpp#5766] [Q&A]
Is it possible to manually increase the text size in the input field used for the Latex editor? The generated output can be changed by increasing the size of the editor dialog, but this does nothing for the input - I often struggle to read the text at its current size.

**Comment by bhennion** [comment]
This would typically be done by changing the global GTK settings and increasing the font sizes therein. The way to do that depends on which OS you're working from.
Otherwise, you could customize your xournalpp.css (its location depends on your platform and OS) and add a line containing `textview { font-size: 8pt; }` inside (replacing 8pt with your desired size, of course).

**Reply by bhennion** [reply]
Actually, there is an option doing exactly that in the Settings dialog, Latex tab, Fonts.
